roles/letsencrypt/tasks/certificates.yml

```diff
@@ -15,19 +15,19 @@
   tags: [letsencrypt_keys]
 
 - name: Generate CSRs for single domain keys
```

Comment:
Let's get rid of this and consolidate the tasks into a single CSR one with the SAN feature. I tested it and it works:

```yaml
- name: Generate CSRs
  shell: "openssl req -new -sha256 -key '{{ letsencrypt_keys_dir }}/{{ item.key }}.key' -subj '/' -reqexts SAN -config <(cat /etc/ssl/openssl.cnf <(printf '[SAN]\nsubjectAltName=DNS:{{ site_hosts | join(',DNS:') }}')) > {{ acme_tiny_data_directory }}/csrs/{{ item.key }}.csr"
  args:
    executable: /bin/bash
    creates: "{{ acme_tiny_data_directory }}/csrs/{{ item.key }}.csr"
  when: site_uses_letsencrypt
  with_dict: "{{ wordpress_sites }}"
  tags: [letsencrypt_keys]
```